src/OidcClient.php

```diff
@@ -99,6 +99,30 @@ public function authenticate(Request $request): OidcTokens
     }
   }
 
+  public function refreshTokens(string $refreshToken): OidcTokens
```

Comment:
This method looks to be mainly a duplication of the already existing authenticate method. I would love to see a refactor so we have as less duplication as possible.

Reply:
@bobvandevijver For me it is ok, the duplication makes sense for me. Moving it into a seperate method would make it harder to read and provides little improvement IMHO.

Reply:
As it is quite a lot of shared implementation, I have now refactored it into a `verifyTokens` method.